### ISSUE TITLE
'ST_NBLOCKSIZE' undeclared

### ISSUE THREAD

**Issue by crscillitoe** [illinoisjackson/even-better-ls#29]
Distribution: Arch

![image](https://user-images.githubusercontent.com/22220179/37559477-75f5a952-29fd-11e8-886d-a179faaba37e.png)

I get this error here when I run
`sh -c "$(curl -fsSL https://raw.githubusercontent.com/illinoisjackson/even-better-ls/master/install.sh)"
`

**Comment by Noammac** [comment]
Did you get the correct coreutils version?

**Comment by crscillitoe** [comment]
I don't see why I wouldn't. I just ran his script I didn't change anything.

**Comment by Noammac** [comment]
If you're using the script you definitely haven't got the correct coreutils
version. The version in the install script is 8.29 while ls.c comes from 8.2

**Comment by crscillitoe** [comment]
So his script is broken? I'll try using coreutils 8.2 I guess

**Comment by Noammac** [comment]
Yeah, Illinois accepted a commit that changed the coreutils version without changing the ls.c version.

There's ls.c for 8.28 in my fork, but that's as modern as it gets AFAIK. Also, it has some issues I hope someone competent would know how to fix.

**Comment by aboutroots** [comment]
In case anyone is interested and if it's OK for you to use an older version, a quick fix to make it to work is to clone the repo and replace the non-working version with 8.2 in the install.sh script before running it.

**Comment by calston** [comment]
So @dkarter broke it?
